applications/hamster.md

```diff
@@ -0,0 +1,278 @@
+
+  Create resource order
+
+  Provider finish order
+
+  Provider heartbeat
+
+  Provider staking amount
+
+  Provider withdraw amount
+
+  Withdraw rental amount
+
+  Withdraw fault execution
+
+  Client cancel order
+
+- Hamster Provider
+
+  Computer create
+
+  Computer destruction
+
+  P2p link listen
+
+- Hamster Client
+
+  List of paid computers
+
+  Computer link usage
+
+- Hamster Front End
+
+  Computing market
+
+  Provider shared computing
+
+  Provider lease agreement list
+
+  List of computers purchased by the customer
+
+  Customer order list
+
+- Hamster Gateway
+
+  Private IPFs network
+
+## Development Roadmap :nut_and_bolt:
+
+### Overview
+
+- **Total Estimated Duration:** ~~3 months~~ 2months
+- **Full-Time Equivalent (FTE):**  4 FTE
+- **Total Costs:** ~~$38,400~~ $25,600
+
+### Milestone 1 Example — Implement Hamster Client and Provider Modules
+
+- **Estimated duration:** 1 month
+- **FTE:**  4
+- **Costs:** $12,800
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0a. | License | Apache 2.0 |
+| 0b. | Documentation | We will provide online documentation of the code and a basic tutorial that includes<br/>1. Hamster Client installation tutorial<br/>2. Hamster Client usage tutorial<br/>3. Hamster Provider installation tutorial<br/>4. Hamster Provider usage tutorial<br/>5. Hamster Chain installation tutorial<br/>6. Hamster Chain usage tutorial |
+| 0c. | Testing Guide | Core functions will be fully covered by unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| 1. | Hamster Client | We will create a desktop client that will have **Market**,**My order** and **My resource** module.<br/>We will provide a desktop (windows, macos) app based on [wails](https://wails.io/) to achieve<br/>1: Market inquiry and management of purchased resources,<br/>2: The point-to-point communication between the client and the computing resource provider.<br/>3: Query historical orders<br/>4: Partial fee refund for defaulted orders<br/>Tech stack: go+wails+go-libp2p+vue.js+polkadot.js |
+| 2. | Hamster Provider | We will create a a resource provider server that will have **Initialize configuration**,**Resource details**,**Account information** and **Configuration information** module.<br/>The resource provider will use the idle resources of the machine, register in the market, and declare that it can provide rental. When the provided virtual machine reaches a transaction in the market, the resource provider provides the virtual machine with the corresponding quota for the remote purchaser to use according to the agreement. The resource provider will establish a p2p connection with the used client for remote management, such as ssh, rdesktop, etc.<br/>Our initial idea is to use the libvirt scheme to implement the management of virtual machines. (Windows uses Hyper-V)<br/>Tech stack: go+go-libp2p+libvirt+go-substrate-rpc-client+gin |
```

Comment:
> Our initial idea is to use the libvirt scheme to implement the management of virtual machines. (Windows uses Hyper-V)

Does that mean, Windows will not be supported? Can you maybe specify the operating systems that will be supported?

Reply:
I apologize for the late reply, I have fixed the problem with the document, thank you very much.